lib/services/local/access_list_test.go

```diff
@@ -125,6 +125,145 @@ func TestAccessListCRUD(t *testing.T) {
 	require.Empty(t, out)
 }
 
+func TestAccessListMembersCRUD(t *testing.T) {
+	ctx := context.Background()
+	clock := clockwork.NewFakeClock()
+
+	backend, err := memory.New(memory.Config{
+		Context: ctx,
+		Clock:   clock,
+	})
+	require.NoError(t, err)
+
+	service, err := NewAccessListService(backend, clock)
+	require.NoError(t, err)
+
+	// Create a couple access lists.
+	accessList1 := newAccessList(t, "accessList1")
+	accessList2 := newAccessList(t, "accessList2")
+
+	cmpOpts := []cmp.Option{
+		cmpopts.IgnoreFields(header.Metadata{}, "ID"),
+	}
+
+	// Create both access lists.
+	accessList, err := service.UpsertAccessList(ctx, accessList1)
+	require.NoError(t, err)
+	require.Empty(t, cmp.Diff(accessList1, accessList, cmpOpts...))
+	accessList, err = service.UpsertAccessList(ctx, accessList2)
+	require.NoError(t, err)
+	require.Empty(t, cmp.Diff(accessList2, accessList, cmpOpts...))
+
+	// There should be no access list members for either list.
+	members, _, err := service.ListAccessListMembers(ctx, accessList1.GetName(), 0, "")
+	require.NoError(t, err)
+	require.Empty(t, members)
+
+	members, _, err = service.ListAccessListMembers(ctx, accessList2.GetName(), 0, "")
+	require.NoError(t, err)
+	require.Empty(t, members)
+
+	// Verify access list members are not present.
+	accessList1Member1 := newAccessListMember(t, accessList1.GetName(), "1")
+	accessList1Member2 := newAccessListMember(t, accessList1.GetName(), "2")
+
+	_, err = service.GetAccessListMember(ctx, accessList1.GetName(), "1")
```

Comment:
Maybe:

```suggestion
	_, err = service.GetAccessListMember(ctx, accessList1.GetName(), accessList1Member1.GetName())
```

Ditto for others?